### PR TITLE
bug 957802 - add Troubleshooting doc

### PR DESCRIPTION
**Pull request by groovecoder**
fix ui-tests headings

move badge below install instructions

### CODE REVIEW

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+       git submodule update --init
+       find . -name "*.pyc" | xargs rm -f
+
+-  If you see ``DatabaseError: (1146, "Table '...' doesn't exist")`` errors,
+   you probably need to run database migrations.::
+
+       python manage.py migrate
+
+   Note: If you are using a vm, this is done when you re-run the Puppet setup.
```
Comment (stephaniehobson) [inline docs/troubleshooting.rst]: I've also had to re-run this in response to errors in vagrant provision that begin with "notice: /Stage[main]/Kuma_config/Exec[kuma_south_migrate]" so vagrant provision does not fix this.

**Comment by stephaniehobson** [comment]
Yay, thanks for doing this.

I had one comment but I don't think it's merge blocking. Once you've seen it we can merge.

File: docs/troubleshooting.rst (hunk 1 above)
Comment (groovecoder) [inline docs/troubleshooting.rst]: Interesting. `provision` certainly runs `migrate`, but if there are errors with `migrate` they will show up as `kuma_south_migrate` errors during provision. I'll add a note to the vagrant troubleshooting section about it.

Comment (groovecoder) [inline docs/troubleshooting.rst]: Added https://github.com/groovecoder/kuma/commit/2b5547f4177ad74771f013bd396eb36637a45a9a for it ...

**Comment by stephaniehobson** [comment]
*kicks Travis*

**Comment by groovecoder** [comment]
Afraid this is a new error for all our builds. :cry: https://github.com/travis-ci/travis-ci/issues/3436

I'm trying to see if I can fix on our side.

**Comment by groovecoder** [comment]
https://github.com/mozilla/kuma/pull/3134 may be the fix (until tox 1.9.2 is released)

**Comment by groovecoder** [comment]
Rebased on master to get the Travis fix.

**Pull request merged**